mail-filter/postforward/postforward-1.0.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit eutils golang-build
+
+DESCRIPTION="Postfix Sender Rewriting Scheme forwarding agent"
+#EGO_PN="github.com/zoni/${PN}"
+EGO_PN="${PN}"
+SRC_URI="https://github.com/zoni/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://github.com/zoni/postforward"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-lang/go:0"
+RDEPEND="
+	mail-filter/postsrsd"
+
+src_prepare() {
+	epatch "${FILESDIR}/${PN}_apply-sendmail-path.patch"
```

Comment:
Define a PATCH array outside this function in which you'll add this file, like so:
```
PATCH=( "${FILESDIR}/${PN}_apply-sendmail-path.patch" )
```